sqlitecluster/SQLite.cpp

```diff
@@ -18,6 +18,8 @@ atomic<int> SQLite::fullCheckpointPageMin(25000); // Approx 100mb (pages are ass
 // Tracing can only be enabled or disabled globally, not per object.
 atomic<bool> SQLite::enableTrace(false);
 
+atomic<sqlite3_stmt*> SQLite::_lastQuery;
```

Comment:
I think this should be a string, and we should just store `sqlite3_normalized_sql((sqlite3_stmt*)p)` in `_sqliteTraceCallback`. Based on the docs ([here](https://www.sqlite.org/c3ref/exec.html) and [here](https://www.sqlite.org/c3ref/stmt.html)), this object is destroyed when `sqlite3_exec` inside `SQuery`, and so we shouldn't be able to access it after `SQuery` returns.

Also, it shouldn't need to be atomic, only one thread has access to this object.